lib/src/builder.dart

```diff
@@ -98,7 +99,11 @@ class GeneratorBuilder extends Builder {
 
 Stream<GeneratedOutput> _generate(LibraryElement unit,
     List<Generator> generators, BuildStep buildStep) async* {
-  for (var element in getElementsFromLibraryElement(unit)) {
```

Comment:
Before, this would just blow up.

Now it'll succeed, but log errors, right? Or are we treating `severe` logs as fatal?

Just trying to understand...